README.md

```diff
@@ -164,6 +164,13 @@ it will run the all the unit test of these project
 - Twitter: [@twitterhandle](https://twitter.com/kamiasif55)
 - LinkedIn: [LinkedIn](https://www.linkedin.com/in/kamran-asif)
 
+
```

Comment:
- [ ] In the project guideline you were asked to screenshot the before and after fixing the N+1 query problem. Kindly edit the PR description and add these screenshots.